dsl/pace/dsl/stencil.py

```diff
@@ -351,6 +377,7 @@ def __init__(
         externals: Optional[Mapping[str, Any]] = None,
         skip_passes: Tuple[str, ...] = (),
         timing_collector: Optional[TimingCollector] = None,
+        comm=None,
```

Comment:
I notice that you default to None here but not for `StencilFactory`. Is it because we don't call `FrozenStencil` directly so we don't expect the users to ever have to see this?

Reply:
Updated StencilFactory.